client/src/cordova/plugin/android/java/org/outline/OutlinePlugin.java

```diff
@@ -209,7 +209,8 @@ private void executeAsync(
           final String tunnelId = args.getString(0);
           final String serverName = args.getString(1);
           final String transportConfig = args.getString(2);
-          sendActionResult(callback, startVpnTunnel(tunnelId, transportConfig, serverName));
+          final String usageReportConfig = args.getString(3);
```

Comment:
No need to have a separate config, it's already included in the `transportConfig`.